gemini/use-cases/retrieval-augmented-generation/intro_multimodal_rag.ipynb

```diff
@@ -351,12 +351,6 @@
     "import urllib.request\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    pdf_path = f"google-10k-sample-14pages.pdf"
</pre><p>This doesn't need to be a f string right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>